src/components/PasswordSetterBox.js

```diff
@@ -56,18 +56,18 @@ class PasswordSetterBox extends Nimiq.Observable {
 
         /* eslint-disable max-len */
         $el.innerHTML = TemplateTags.hasVars(1)`
-            <div class="password-strength strength-short  nq-text-s" data-i18n="passwordbox-password-strength-short" >Enter at least 8 characters</div>
-            <div class="password-strength strength-weak   nq-text-s" data-i18n="passwordbox-password-strength-weak"  >That is a weak password</div>
-            <div class="password-strength strength-good   nq-text-s" data-i18n="passwordbox-password-strength-good"  >Ok, that is an average password</div>
-            <div class="password-strength strength-strong nq-text-s" data-i18n="passwordbox-password-strength-strong">Great, that is a strong password</div>
-            <div class="password-strength strength-secure nq-text-s" data-i18n="passwordbox-password-strength-secure">Super, that is a secure password</div>
+            <div class="password-strength strength-short  nq-text-s" data-i18n="passwordbox-password-strength-short" >Enter min. 8 characters</div>
+            <div class="password-strength strength-weak   nq-text-s" data-i18n="passwordbox-password-strength-weak"  >Sufficient password</div>
```

Comment:
Suggestion to change this to
```
Weak password
```
to make clear that it is _weak_, and not actually sufficient in the interpreted meaning of the word.
@julianbauer

Reply:
Weak could be interpreted as not sufficient, while it is. But as the potential 'error' happening here is a user having to secure their PW further, let's go with your suggestion.